Sign_up/sign_up.php

```diff
@@ -0,0 +1,723 @@
+<?php
+
+session_start();
+
+include("./connect_db.php");
+
+if (isset($_POST['button']) && isset($_POST['email'])) {
+
+    $email = filter_var($_POST['email'], FILTER_SANITIZE_EMAIL);
+
+    if (filter_var($email, FILTER_VALIDATE_EMAIL)) {
+
+        // Check exist email
+        $select_email = "SELECT `email` FROM `user` WHERE `email` = '$email'";
+        $check_email = mysqli_query($conn, $select_email);
+
+        if (mysqli_num_rows($check_email) != 0) {
+            echo "<script>alert('This email was being used')</script>";
+        } else {
+            $_SESSION['email'] = $email;
+            header("Location: http://localhost/FOOD_STORE_WEBSITE/sign_up/verify_pass.php");
+            die();
+        }
+    } else {
+        echo "<script>alert('Email invalid');</script>";
+    }
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+
+    <link rel="apple-touch-icon" type="image/png" href="https://cpwebassets.codepen.io/assets/favicon/apple-touch-icon-5ae1a0698dcc2402e9712f7d01ed509a57814f994c660df9f7a952f3060705ee.png" />
+    <meta name="apple-mobile-web-app-title" content="CodePen">
+
+
+    <link rel="mask-icon" type="image/x-icon" href="https://cpwebassets.codepen.io/assets/favicon/logo-pin-b4b4269c16397ad2f0f7a01bcdf513a1994f4c94b8af2f191c09eb0d601762b1.svg" color="#111" />
+    <title>Mailbox Submit</title>
+</head>
+
+<body translate="no">
+    <script src="../assets/js/verify_pass.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://kit.fontawesome.com/bf5ba39767.js" crossorigin="anonymous"></script>
+
+    <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+
+    <div id="svgWrapper">
```

Comment:
This code is not readable, you can move the sgv to a separated file or just remove it